sim/plots.py

```diff
@@ -0,0 +1,24 @@
+import matplotlib
```

Comment:
I there should be a sub-directory for plots

Reply:
ah ok, I will put a todo for that.

Reply:
also going to put this code within the `fold_coverage_test` module